opencga-catalog/src/main/java/org/opencb/opencga/catalog/migration/MigrationTool.java

```diff
@@ -249,6 +252,29 @@ protected final void queryMongo(String inputCollectionStr, Bson query, Bson proj
         }
     }
 
+    protected final <T> List<List<T>> generateBulks(List<T> list) {
```

Comment:
Could this be replaced with BatchUtils?
https://github.com/opencb/opencga/blob/develop/opencga-storage/opencga-storage-core/src/main/java/org/opencb/opencga/storage/core/utils/BatchUtils.java#L31

Reply:
Yes, should we move that BatchUtils class somewhere else? Generating those bulks of data is super useful.

Reply:
Yes, definitely, it could be moved either to opencga-core or to java-commons-lib
Feel free to move it

Reply:
Moving it to core